### ISSUE TITLE
local muxrpc validation error, crash

### ISSUE THREAD

**Issue by qypea** [ssbc/scuttlebot#527]
My sbot pub started crashing when I updated to the latest npm versions today, with the following stack trace. It seems to likely be an issue with schema validation somewhere, but hard for the uninitiated to be sure.

```
 /home/pi/easy-ssb-pub/node_modules/muxrpc-validation/index.js:59
       if (err) throw err
                ^
 UsageError: Param 0 is required
     at Error.ZE (/home/pi/easy-ssb-pub/node_modules/zerr/index.js:11:42)
     at Object.ZError [as MissingParam] (/home/pi/easy-ssb-pub/node_modules/zerr/index.js:13:14)
     at validate (/home/pi/easy-ssb-pub/node_modules/muxrpc-validation/index.js:122:48)
     at Object.<anonymous> (/home/pi/easy-ssb-pub/node_modules/muxrpc-validation/index.js:58:17)
     at Object.<anonymous> (/home/pi/easy-ssb-pub/node_modules/secret-stack/util.js:80:17)
     at Object.<anonymous> (/home/pi/easy-ssb-pub/node_modules/hoox/index.js:3:17)
     at Object.hooked [as add] (/home/pi/easy-ssb-pub/node_modules/hoox/index.js:10:15)
     at Socket.<anonymous> (/home/pi/easy-ssb-pub/node_modules/scuttlebot/plugins/local.js:59:21)
     at emitOne (events.js:96:13)
     at Socket.emit (events.js:188:7)
```

**Comment by cryptix** [comment]
This looks like https://github.com/ssbc/multiserver/pull/16 - Can you run `npm ls multiserver` in your easy-ssb-pub installation and check if the version is older than v1.13.3?

**Comment by qypea** [comment]
Well, it is v1.13.3 but I haven't been able to reproduce the crash this afternoon. Will monitor overnight and see if it happens again.

**Comment by staltz** [comment]
It should be fixed by multiserver@1.13.3. Let's close this and we can open another issue if something else occurs.
